src/utils/ucc_byte_order.h

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020-2021.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+//TODO was taken from here, rights issue?
+// https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/platform/byte_order.h
```

Comment:
Yes. You need to include the Apache license. There is some information on how to do this here:
https://www.apache.org/licenses/LICENSE-2.0